modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/js/FileEntryPicker.js

```diff
@@ -22,20 +22,39 @@ import React, {useEffect, useState} from 'react';
 
 const FileNamePicker = ({
 	maxFileSize: initialMaxFileSize,
+	maxMimeTypeSize,
 	namespace,
 	validExtensions,
 }) => {
-	const maxFileSize = Number(initialMaxFileSize);
+	let maxFileSize = Number(initialMaxFileSize);
 	const inputId = `${namespace}file`;
 	const [inputValue, setInputValue] = useState('');
 	const [fileName, setFileName] = useState('');
 	const [maxFileSizeError, setMaxFileSizeError] = useState(false);
+	const [maxSize, setMaxSizeValue] = useState(maxFileSize);
```

Comment:
Update `setMaxSizeValue` to `setMaxSize`.

```suggestion
	const [maxSize, setMaxSize] = useState(Number(initialMaxFileSize));
```

Reply:
I removed it because it is not used after source code is updated.